docs/php/versioning.md

```diff
@@ -95,7 +95,7 @@ if ($v === 1) {
 
 You'll note that `minSupported` has changed from `DEFAULT_VERSION` to `1`. If an older version of the
 Workflow Execution history is replayed on this code, it will fail because the minimum expected version
-is 1. After you are sure that all of the Workflow executions for version 1 have completed, then you
+is 1. After you are sure that all of the Workflow Executions for version 1 have completed, then you
 can remove 1 so that your code would look like the following:
```

Comment:
```suggestion
After you are sure that all Workflow Executions for version 1 have completed, you can remove version 1 so that your code looks like the following:
```